corehq/apps/locations/views.py

```diff
@@ -821,7 +821,6 @@ def _get_manage_consumption():
         })
         return context
 
-    @method_decorator(lock_locations)
```

Comment:
What's the difference in behavior here?  Seems like the user should get a hard failure if they try to submit overlapping imports.

Reply:
`import_locations_async` uses the `serial_task` decorator, so it will retry if the lock is not acquired.  I'm not sure that's actually desirable.  My read of this is that that functionality doesn't actually matter under the current behavior, because this lock prevented users from queuing multiple tasks anyways.  You could probably make an argument for either behavior, I just want to make sure we're doing it deliberately.  What do you think?

Side note: I just recently implemented similar logic for saved export rebuilds: 
https://github.com/dimagi/commcare-hq/blob/e6f7253acfafa1c33cc2184925d7dbcd01f5a1ae/corehq/apps/export/tasks.py#L97-L102

Reply:
> `import_locations_async` uses the `serial_task` decorator, so it will retry...

I did not realize it will retry, and I agree that does not seem desirable. I'll change it.

Reply:
https://github.com/dimagi/commcare-hq/pull/21170/commits/77b0e1afa880864ef2605be2c31ba8906bb1fce5